react/login.tsx

```diff
@@ -1,25 +1,44 @@
-import React from "react";
-
-import useLogin from '/react/api/useLoggin';
-
-import Button from '.'
-import Input from '.'
+import React, {useState, useEffect} from "react";
+import useLogin from '/react/api/useLoggin.ts';
+import Button from './Button.ts'
+import Input from './Input.ts'
 
 function Login() {
+  const { submitLogin } = useLoggin()
+  const [username, setUsername] = useState('')
+  const [password, setPassword] = useState('')
+  const [valid, setValid] = useState(false)
 
-  return (<div>
-    <Input 
-      value={'username'} 
-      onChange={() => {}}
-    />
-    <Input
-      value={'password'}
-      secret
-    />
-    <Button onClick={() => {}} disabled>
-      Submit
-    </Button>
-  </div>
+  useEffect(() => {
+    // if inputs are non empty -> valid
+    if(username.length && password.length) {
+      setValid(!valid)
+    }
+  }, [valid])
```

Comment:
I believe there's a mistake here... should be `[username, password]` instead of `[valid]`